.github/workflows/static-checks.yaml

```diff
@@ -0,0 +1,74 @@
+on: ["pull_request"]
+name: Static checks
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x, 1.15.x]
+        os: [ubuntu-18.04]
+    runs-on: ${{ matrix.os }}
+    env:
+      GO111MODULE: off
+      TRAVIS: "true"
+      TRAVIS_BRANCH: ${{ github.base_ref }}
+      TRAVIS_PULL_REQUEST_BRANCH: ${{ github.head_ref }}
+      TRAVIS_PULL_REQUEST_SHA : ${{ github.event.pull_request.head.sha }}
```

Comment:
Which scripts rely on these variables? Would it makes sense to get rid of it, since it's no longer travis ?

Reply:
@mxpv the static-checks.sh scripts, we need those variables to make it work currently as somebody needs to change the scripts to support github actions (variables, setup, and so on)

Reply:
I can't find anything that use travis variables in `ci` directory (am I missing something?)

Reply:
@mxpv here it is https://github.com/kata-containers/tests/blob/2.0-dev/.ci/static-checks.sh, it is also being used at the check-commits.sh script

Reply:
Got it, thanks for clarification.